README.md

```diff
@@ -17,48 +17,31 @@ Once in a while, it connects to APIs to get the latest information about session
 It then augments and republishes this information for clients to consume. 
 It also provides a couple of extra APIs to save your favorites and accumulate votes.
 
-### Android Application
+### iOS and Android Applications
 
-As you can imagine, the Android version is developed in Kotlin/JVM. What's interesting here is that this time
-application utilizes Multiplatform support, which is an experimental feature in Kotlin 1.3.61. Data structures for retrieving data from the backend server and some date-time operations are shared across multiple projects.
-
-### iOS Application
-
-User interface of iOS version is written in Swift, all logic and data written in Kotlin in the common module. 
-This way iOS part itself is responsible only for specifying how the application looks like and how it represents changes requested by logic. Kotlin and Swift are highly interoperable, so from Swift, you can easily use all classes and tools defined in the common module. 
+Both Android and iOS applications are developed using Kotlin Multiplatform Mobile technology within a single codebase.
```

Comment:
It would be nice to include links here.

Edit + Links: The Android and iOS applications are developed within a single codebase using [Kotlin Multiplatform technology](https://kotlinlang.org/docs/multiplatform.html). The UI is shared between both platforms by using the [Compose Multiplatform UI framework](https://www.jetbrains.com/lp/compose-multiplatform/).

Reply:
Please also mention Web version

Reply:
sure, will do